doc/PROTOCOL-HTTP2.md

```diff
@@ -56,6 +57,23 @@ If **Timeout** is omitted a server should assume an infinite timeout. Client imp
 
 Note that HTTP2 does not allow arbitrary octet sequences for header values so binary header values must be encoded using Base64 as per https://tools.ietf.org/html/rfc4648#section-4. Implementations MUST accept padded and un-padded values and should emit un-padded values. Applications define binary headers by having their names end with "-bin". Runtime libraries use this suffix to detect binary headers and properly apply base64 encoding & decoding as headers are sent and received.
 
+**Custom-Metadata** header order is not guaranteed to be preserved except for
+values with duplicate header names. Duplicate header names may have their values
+joined with "," as the delimiter and be considered semantically equivalent.
+Implementations must split **Binary-Header**s on "," before decoding the
+Base64-encoded values.
+
+**ASCII-Value** should not have leading or trailing whitespace. If it contains
+leading or trailing whitespace, it may be stripped. The **ASCII-Value**
+character range defined is more strict than HTTP. Implementations must not error
+due to receiving an invalid **ASCII-Value** but value valid in HTTP, but the
```

Comment:
The wording is awkward because **ASCII-Value** does not exist in HTTP, so it by itself can't be in HTTP.

I think I swapped two words. Does this work better for you @jcanizales:
> Implementations must not error due to receiving an invalid **ASCII-Value** _that's a valid value_ in HTTP, but the precise behavior is not strictly defined

Even better, maybe it should be **field-value** since that the precise term in HTTP:
> Implementations must not error due to receiving an invalid **ASCII-Value** _that's a valid **field-value**_ in HTTP, but the precise behavior is not strictly defined